Gruntfile.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+module.exports = function (grunt) {
+
+    // TODO:  Clean up the project structure into cleanly separated pieces with their own standard src and tests.
+    // TODO:  Set up separate checks for the couchapp content, with a different .jshintrc
+    grunt.initConfig({
+        jshint: {
+            src: ["./**/*.js", "email/**/*.js", "front-end/**/*.js", "import/**/*.js", "test/**/*.js"],
```

Comment:
Sadly, the original was from one of my earliest and sloppiest efforts.  I cleaned things up.